preview.py

```diff
@@ -0,0 +1,21 @@
+import sys
```

Comment:
Would it perhaps be good to have this (or something with a distinctive enough name) added to the path after you pip install ricecooker? Maybe a `ricecooker` command with subcommands like `ricecooker preview ...`? Since most people aren't cloning the source, it's otherwise a pain to find this file in the installed packages etc in order to run it.

Reply:
I see this approach as more of a band-aid for internal use than a solution we want to commit to, so I didn't go too far with it. I think the ultimate fix for this will be two-fold:

1. For content developers: A working Studio -> Kolibri preview / test workflow, maybe from dev boxes so we don't affect Kolibri proper.
2. For Kolibri / Studio devs: Working integration tests, so that we can not only test that it works, but that it keeps working day in and day out, like the Energizer Bunny. Even when you update Chrome. ;)

I'd only want to really commit to this route and polish it into something easy and accessible once I know that #2 is not really viable, but right now I still think there's a solution to be had there. This just helps us to keep making progress on the HTML5 front until we get there.